src/iii_conventions/MyDate.kt

```diff
@@ -1,5 +1,10 @@
 package iii_conventions
 
+/**
+ * Pre-defined classes for tasks in iii_conventions
+ * DO NOT CHANGE THIS FILE.
```

Comment:
I don't get why you'd tell not to modify this file as you have to implement several things in it (compareTo() etc...). Do you have something else in mind ?